orc8r/cloud/deploy/terraform/orc8r-helm-aws/examples/online-upgrade/variables.tf

```diff
@@ -143,21 +143,22 @@ variable "elasticsearch_domain_configuration" {
 variable "docker_registry" {
   description = "URL to your Docker registry"
   type        = string
+  default     = "https://docker.artifactory.magmacore.org/artifactory/docker"
 }
```

Comment:
Has anyone validated the "online-upgrade" example in a while? If not we can just delete it for now

Reply:
Will delete it as part as another PR, created an issue there https://github.com/magma/magma/issues/8126